docs/03-tutorials/00-api-exposure/apix-09-expose-and-secure-workload-with-certificate.md

```diff
@@ -0,0 +1,291 @@
+---
+title: Expose and secure a workload with a certificate 
+---
+
+This tutorial shows how to expose and secure a workload with mutual authentication using Kyma's mutual TLS Gateway. 
+
+## Prerequisites
+
+This tutorial is based on a sample HttpBin service deployment and a sample Function. To deploy or create one of those, follow the [Create a workload](./apix-01-create-workload.md) tutorial.
+
+Before you start, Set up [`kyma-mtls-gateway`](../00-security/sec-02-setup-mtls-gateway.md) to allow mutual authentication in Kyma and make sure that you exported the [bundle certificates](../00-security/sec-02-setup-mtls-gateway#steps). 
+
+## Expose and access your workload
+
+Follow the instruction to expose and access your instance of the HttpBin service or your sample Function.
+
+<div tabs>
+  <details>
+  <summary>
+  HttpBin
+  </summary>
+
+1. Export the following values as environment variables:
+
+     ```bash
+   export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME}
+   export GATEWAY=$NAMESPACE/$MTLS_GATEWAY_NAME
+   export CLIENT_CERT_CRT_FILE=client.example.com.crt
```

Comment:
fix formatting, use placeholders to avoid using client.example.com. @dariusztutaj can you propose something generic for CN and O?